### DISCUSSION TITLE
libri light dataset

### DISCUSSION THREAD

**Discussion by loganhart02** [coqui-ai/TTS#698] [Ideas]
Hey guys! I was looking for some asr datasets for a conformer model and I stumbled upon the libri light dataset and I think it could be something to use to help train a speaker encoder and wanted to get your thoughts or if you've seen it yet. The main dataset is 60k hours of unlabeled audio with speaker ID labels. I'm not sure if we need the audio transcripts for speaker encoder training but my intuition is that you only need the audio and the speaker Id. the main dataset has 7439 speakers.
Here is the link to the dataset https://github.com/facebookresearch/libri-light
the link to the paper https://arxiv.org/pdf/1912.07875.pdf
and Ill drop a screenshot below of the different dataset sizes
![Screenshot from 2021-07-30 20-03-10](https://user-images.githubusercontent.com/72301874/127940467-febe9b48-a9ef-4a06-a4d2-321f388ce547.png)

**Comment by erogol** [comment]
Thanks for the link. I think @Edresson  trained a speaker encoder (yet to release) on that

**Comment by AbubakarAliyuBadawi** [comment]
@loganhart02 Please do you know how I can get the transcript of the Libri Light large 60k hours? I want to see if synthesizing the dataset will improve the semantic performance of my SLM

**Comment by loganhart02** [comment]
look for libriheavy dataset it's libri light but with GT text from the
actual books
